content/kapp-controller/_index.html

```diff
@@ -3,49 +3,58 @@
-        </div>
-        <div class="col">
-            <div class="icon">
-                <img src="/img/modularized-config.svg" />
-            </div>
-            <h3>Composable</h3>
-            <p>Take advantage of different tools for each stage -- fetch, template and deploy. For example, fetch Helm templates from a Git repository, use Helm to template them, make modifications with ytt overlays, and finally deploy with kapp.</p>
-        </div>
-        <div class="col">
-            <div class="icon">
-                <img src="/img/deterministic.svg" />
-            </div>
-            <h3>Lightweight</h3>
-            <p>kapp-controller delegates all the heavy lifting to existing configuration management and deployment tools. In doing so, kapp-controller itself is kept very simple, and therefore provides a high degree of reliability.</p>
-        </div>
+  <div class="yaml-grid grid three">
+    <div class="col">
+      <div class="icon">
+        <img src="/img/structure-oriented.svg" />
+      </div>
+      <h3>Continuous delivery</h3>
+      <p>kapp-controller's App CR provides a consistent API for fetching, templating, deploying, and syncing software from an underlying 
+        source out to a Kubernetes cluster, regardless if it is a Helm chart, plain YAML configuration, ytt templates, etc.</p>
     </div>
+    <div class="col">
+      <div class="icon">
+        <img src="/img/modularized-config.svg" />
+      </div>
+      <h3>Package management</h3>
+      <p> kapp-controller's package management API allows users to encapsulate, version, and share software packages via a Kubernetes cluster as 
+        Kubernetes custom resources. By making software available as Package CRs, package consumers can discover, configure, and install software 
+        on Kubernetes all through a simple interface.</p>
+    </div>
+    <div class="col">
+      <div class="icon">
+        <img src="/img/deterministic.svg" />
+      </div>
+      <h3>Lightweight, flexible</h3>
+      <p>kapp-controller delegates all the heavy lifting to existing configuration management and deployment tools. In
+        doing so, kapp-controller itself is kept very simple, and therefore provides a high degree of reliability. While 
+        simple and reliabile, kapp-controller supports several use cases around deploying and managing software on Kubernetes.</p>
+    </div>
+  </div>
 </div>
 
 <div class="wrapper">
-    <div class="text-content" id="examples">
-        <h2>Basic Usage</h2>
-        <p>Describe in <code>simple-app.yml</code> how to fetch, template, and deploy your application:</p>
-        {{< highlight yaml "" >}}
+  <div class="text-content" id="examples">
+    <h2>Basic Usages</h2>
+    <h3>Continuous delivery</h3>
+    <p>Want something lightweight, easy to start with, still providing flexibility in how to fetch, template, and deploy for your 
```

Comment:
> Want something lightweight, easy to start with, still providing flexibility in how to fetch, template, and deploy for your software? Use kapp-controller's App CR.

Suggestion:
Want something lightweight and easy to start with that still provides flexibility in the tools you choose to fetch, template, and deploy? Check out app-controller's App CR.